src/strings/PowerFxResources.en-US.resx

```diff
@@ -530,6 +530,28 @@
   <data name="AboutBooleanNT_input" xml:space="preserve">
     <value>A column of number values to process.</value>
   </data>
+  <data name="AboutBooleanB" xml:space="preserve">
+    <value>Converts a 'boolean' to a boolean value.</value>
+    <comment>Description of 'Boolean' function.</comment>
+  </data>
+  <data name="BooleanBArg1" xml:space="preserve">
+    <value>boolean</value>
+    <comment>function_parameter - First argument to the Boolean function - the boolean that will be converted to a boolean.</comment>
+  </data>
+  <data name="AboutBooleanB_boolean" xml:space="preserve">
+    <value>The boolean value to convert to a boolean value.</value>
```

Comment:
>he boolean value to convert to a boolean value. [](http://example.com/codeflow?start=12&length=47)

nit - we should get updated string.  This doesn't actually convert - it's just included for stylist